dotCMS/src/main/webapp/html/portlet/ext/folders/menu_actions_js.jsp

```diff
@@ -591,14 +593,17 @@
 	}
 
 	function editContentlet (objId, userId, referer, live, working, write) {
-        var customEvent = document.createEvent("CustomEvent");

```

Comment:
Did you try to listen for the double click event and prevent it? https://developer.mozilla.org/en-US/docs/Web/API/Element/dblclick_event

Reply:
mmm, when trying to prevent the `double click` event, the `click` event will execute first.

https://codesandbox.io/s/frosty-bhaskara-dg20p